quinn-proto/src/frame.rs

```diff
@@ -633,7 +637,7 @@ impl Iter {
                 Frame::Ack(Ack {
                     delay,
                     largest,
-                    additional: self.bytes.get_ref().slice(start..end),
+                    additional: Bytes::copy_from_slice(&self.bytes.get_ref()[start..end] as &[u8]),
```

Comment:
looks like another new deep copy?

I don't think it should do this. If it already has to allocate, it should at least immediately decode the ACKs blocks and have those stored in a list, so that this work goes away later on.